.github/workflows/publish.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+
+    env:
+      CF_API_KEY: ${{ secrets.CF_API_KEY }}
+      GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
+      WAGO_API_TOKEN: ${{ secrets.WAGO_API_TOKEN }}
+      WOWI_API_TOKEN: ${{ secrets.WOWI_API_TOKEN }}
```

Comment:
`GITHUB_TOKEN` magically exists, but the remaining tokens need to be added. Until they are added, the action will just ignore them. Until `CF_API_KEY` is added, localisations will be broken though, so I suggest only creating the first release tag once that's entered. Merging before that is fine as that won't trigger the action yet.